src/api/todos.ts

```diff
@@ -0,0 +1,24 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const USER_ID = 29;
+
+export const getTodos = () => {
+  return client.get<Todo[]>(`/todos?userId=${USER_ID}`);
+};
+
+export const getDelete = (todoId: number) => {
+  return client.delete(`/todos/${todoId}`);
+};
+
+export const getAdd = ({ title, completed, userId }: Omit<Todo, 'id'>) => {
+  return client.post<Todo>('/todos', { title, completed, userId });
+};
+
+export const getUpdate = (
+  id: Todo['id'],
+  title: Todo['title'],
+  completed: Todo['completed'],
+) => {
+  return client.patch<Todo>(`/todos/${id}`, { title, completed });
+};
```

Comment:
Please, fix function names and make them consistent
For instance:
getTodos, deleteTodo, addtodo, updateTodo

Reply:
Also, add and update methods are inconsistent in terms of parameters:
- add has one parameter of the type `Omit<Todo, 'id'>)`
- update has three primitive parameters

Please either convert update to accept single parameter, or add to accept set of the primitives